packages/react-server/core/config.js

```diff
@@ -12,14 +12,14 @@ if (SERVER_SIDE) {
 		// only read out the config once, and then cache it. -sra.
 		if (null === config) {
 			/*eslint-disable no-process-env */
-			if (!process.env.TRITON_CONFIGS) {
+			if (!process.env.REACT_SERVER_CONFIGS) {
 				/*eslint-enable no-process-env */
-				throw new Error('TRITON_CONFIGS environment variable required to start server.');
+				throw new Error('REACT_SERVER_CONFIGS environment variable required to start server.');
```

Comment:
I'm not opposed to the `TRITON_CONFIGS` --> `REACT_SERVER_CONFIGS` change, but since it breaks all clients, I probably wouldn't normally roll it into an examples PR.